helix-view-aggregator/src/main/java/org/apache/helix/view/aggregator/HelixViewAggregator.java

```diff
@@ -75,7 +75,7 @@ public HelixViewAggregator(String viewClusterName, String zkAddr) {
     _viewClusterManager = HelixManagerFactory
         .getZKHelixManager(_viewClusterName, generateHelixManagerInstanceName(_viewClusterName),
             InstanceType.SPECTATOR, zkAddr);
-    _refreshViewCluster = new AtomicBoolean(false);
+    _refreshViewCluster = new AtomicBoolean(true);
```

Comment:
why change default from 'false' to 'true'

Reply:
I think it makes more sense to trigger a refresh after startup (in the upcoming periodic refresh)